libs/sync/src/assets/i18n/en.json

```
@@ -1,109 +1,138 @@
+	"sync.details.settings.modification-date.from": "From",
+	"sync.details.settings.modification-date.label": "Modification date filter",
+	"sync.details.settings.modification-date.to": "to",
+	"sync.details.settings.name": "Name",
+	"sync.details.settings.synchronize-security-groups": "Synchronize the security groups",
+	"sync.details.settings.title": "Connector settings",
+	"sync.details.tabs.folders": "Folders",
+	"sync.details.tabs.settings": "Settings",
+	"sync.details.toast.activating-failed": "Activating the sync failed",
+	"sync.details.toast.deletion-failed": "Sync deletion failed",
+	"sync.details.toast.disabling-failed": "Disabling the sync failed",
+	"sync.details.toast.modification-failed": "Modifying sync failed",
+	"sync.details.toast.modification-success": "Sync modified successfully",
+	"sync.details.toast.triggering-sync-failed": "Sync triggering failed",
+	"sync.details.toast.triggering-sync-success": "Sync successfully triggered",
+	"sync.folder-selection.edit-folders-button": "Edit folders selection",
+	"sync.folder-selection.empty-message": "You didn't select any folder yet",
+	"sync.folder-selection.filtered-empty-message": "No folder match your query",
+	"sync.folder-selection.footer.save-changes": "Save changes",
+	"sync.folder-selection.footer.selection-count": "{{count}} folders selected",
+	"sync.folder-selection.search-input-label": "Search a folder title",
+	"sync.folder-selection.see-all": "See all ({{total}})",
+	"sync.folder-selection.see-selection": "See selected ({{count}})",
+	"sync.form-validation.extra-parameters-incomplete": "Incomplete rows are ignored",
+	"sync.home-page.add-sources.description": "Automatically push content from any source to your current Knowledge Box",
+	"sync.home-page.add-sources.disabled-source-tooltip": "Activate a sync server",
+	"sync.home-page.add-sources.title": "Add sources",
+	"sync.home-page.description": "Upload and synchronize content from third-party sources.",
+	"sync.home-page.setup-server.description": "The Nuclia Sync Agent is an application that allows you to synchronize a local folder or a third-party data source. Its purpose is to manage all the 3rd-party storage services authentication and data transfer on your premises for privacy and security reasons.",
+	"sync.home-page.setup-server.on-desktop.download-dropdown": "Download",
+	"sync.home-page.setup-server.on-desktop.radio-help": "To install the sync server locally, <a href=\"https://github.com/nuclia/sync-agent/releases\" target=\"_blank\">download</a> the latest version for your system.",
+	"sync.home-page.setup-server.on-desktop.radio-label": "Use the Nuclia Sync Agent on your desktop",
+	"sync.home-page.setup-server.on-server.radio-help": "To deploy it manually on a server, follow these <a href=\"https://github.com/nuclia/sync-agent/blob/main/server/README.md\" target=\"_blank\">instructions</a>.",
+	"sync.home-page.setup-server.on-server.radio-label": "Deploy it manually on a server",
```

I think "Deploy" is not accurate (the user will deploy it once, but then they will use the server, not deploy it every time).
```suggestion
	"sync.home-page.setup-server.on-server.radio-label": "Use a remote server",
```